File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    subArr = list(map(str, s.split(" ")))
    arr = [0]*len(subArr)
    for i in range(len(arr)):
        arr[i] = int(subArr[i])
    
    answer = str(min(arr)) + " " + str(max(arr))
    return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.io.*;
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] str = s.split(" ");
        ArrayList<Integer> arr = new ArrayList<>();
        for(int i=0; i<str.length; i++) {
            arr.add(Integer.parseInt(str[i]));
        }
        
        Collections.sort(arr);
        int min = arr.get(0);
        int max = arr.get(arr.size() - 1);
        
        return min + " " + max;
    }
}